doc/mediasdk-man.md

```diff
@@ -1973,17 +1973,23 @@ This function initiates execution of an asynchronous function not already starte
 | | |
 --- | ---
 `MFX_ERR_NONE` | The function completed successfully.
+`MFX_ERR_NONE_PARTIAL_OUTPUT` | The function completed successfully, bitstream contains a portion of the encoded frame according to required granularity.
 `MFX_WRN_IN_EXECUTION` | The specified asynchronous function is in execution.
 `MFX_ERR_ABORTED` | The specified asynchronous function aborted due to data dependency on a previous asynchronous function that did not complete.
 
 **Change History**
 
 This function is available since SDK API 1.0.
+Return code MFX_ERR_NONE_PARTIAL_OUTPUT is introduced in SDK API TBD.
 
 **Remarks**
 
 See status codes for specific asynchronous functions.
 
+In partial output mode, enabled during encoder initialization [mfxExtPartialBitstreamParam](#mfxExtPartialBitstreamParam), application is expected to call SyncOperation repeatedly until it receives the MFX_ERR_NONE (or some error);
+when MFX_ERR_NONE_PARTIAL_OUTPUT code is returned, new portion of the encoded frame is located at DataOffset location and have a DataLength size in the associated [mfxBitstream](#mfxBitstream) buffer.
```

Comment:
This blows out my mind. So, I called SyncOperation for the first time. And while I am accessing first portion library might change DataOffset to point to the next portion? The behavior you describe is direct description of the race condition. Please, clarify/change. Also, please, provide code example right in the manual.

Reply:
As commented before. There is nothing special for this feature. The same behavior can happen in normal operation mode when SyncOperation returns IN_EXECUTION status.